includes/modules/featured_categories.php

```diff
@@ -63,7 +63,7 @@
         $featured_categories->MoveNextRandom();
     }
 
-    if (!$featured_categories->EOF) {
+    if ($featured_categories->EOF) {
```

Comment:
Right, having traversed the items retrieved from the database, the returned database object will now be at `EOF`.  Given that this code section is already within a clause that's not going to be run if there aren't any featured categories, I'll suggest just removing the check on line 66 (and its closing brace) and sliding the code presently in that section to align with the ending brace for the `while` loop above.